frontend/src/app/middleware/middleware.service.ts

```diff
@@ -83,7 +83,7 @@ export class MiddlewareService {
       .subscribe(
         success => {
           // Log when successful / errored navigation
-          this.router.navigate([`/claims/${claim.vehicle.vin}`]).then(
+          this.router.navigate([`/claims`]).then(
```

Comment:
You can probably undo this as well - in my PR i have a little dumb workaround for the data not showing.

Reply:
I left it for the time being so my branch is still functional

Reply:
Revert this - my branch fixes it temporarily